classes/class4/README.md

```diff
@@ -1 +1,484 @@
+* [`<form>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form):
+Defines a section of interactive controls and inputs that will submit information to a server.
+These will usually include a number of [`<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input)
+tags.
+* [`<h1>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/h1):
+Header text. Pretty common, there are six tags h1 - h6. Use these rather than a `<p>` with a css class `header-1` or something like that.
+* [`<img>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img):
+Display an image with the image tag. `<img>` tags cannot have children and do not have an end tag (`<img source="./foo.jpg" />`).
+* [`<ul>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul),
+  [`<ol>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol), and
+  [`<li>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li):
+Un-ordered list, ordered list, and list element tags are used for structuring list content in HTML.
+* [`<p>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p):
+The paragraph element defines a block of text.
+* [`<span`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span):
+The span element is like a div in that it is a generic container element.
+However, in contrast, spans are inline elements by default (they do not break the flow of elements on the page)
+and are often used to style text: `<p><span class="fancy-text">Some text</span></p>`.
+* [`<table>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table):
+Prior to the use of CSS, the table element, along with `<th>` (table header), `<tr>` (table row), and `<td>` (table data),
+were often used as a method for page layout. Please don't do this.
+They can still be used for the categorization of data,
+but the styling and layout or your pages should all be controlled by CSS.
+
+Check out this [site](http://www.htmldog.com/guides/css/intermediate/layout/)
+or Google "HTML layout tips" to read more about formatting techniques.
+
+### JSFiddle
+Check out [JSFiddle](http://jsfiddle.net/vsw1vLz6/).
+It is a super useful tool that allows you to test small snippets of HTML, CSS, and JavaScript and see the output.
+You can also share and do some other cool stuff with it.
+
+## HTML Forms
+In the last homework, we passed information to the server through the URL.
+This required us to do everything as a GET request, which is BAD, and was pretty limiting.
+Of course it was also a pretty atrocious User Experience (UX).
+```
+GET /cats/new
+GET /cats
+GET /cats/bycolor/:color
+GET /cats/delete/old
+```
+Now we're going to look at a better way to do this using HTML forms.
+
+To start we'll need a basic Express app again.
+You can start a new app or modify your homework, but we'd recommend starting from last class's app.
+From the class4 folder:
+```
+cp -r ../class3/expressintro ./formsApp
```

Comment:
`::expressintro::` or some other kind of formatting to differentiate it? and same on line below.